armi/nuclearDataIO/xsCollections.py

```diff
@@ -381,6 +381,12 @@ def __init__(
     def createMacrosOnBlocklist(
         self, microLibrary, blockList, nucNames=None, libType="micros"
     ):
+        """Create macroscopic cross sections for a list of blocks.
+
+        .. impl:: Build macroscopic cross sections for blocks.
+            :id: I_ARMI_MACRO_XS
+            :implements: R_ARMI_MACRO_XS
```

Comment:
So this requirement looks like it's supposed to live in the macroXSGenerationInterface. Based on that, it looks like this is probably the intended implementation:
https://github.com/terrapower/armi/blob/0c4d5f4c4353f670ce6096b239af360addcc6fde/armi/physics/neutronics/macroXSGenerationInterface.py#L131-L140

This method in xsCollections.py looks relatively unused, actually.

Reply:
it is not used in ARMI, but it _is_ used downstream, but some internal projects.

So, there's that. Hmmm

Reply:
Actually, that class `MacroXSGenerationInterface` is used  heavily, though only by one downstream project.

Which is interesting, and maybe we could put a not on that  project to see if that should be moved out of ARMI. 

But that file isn't part of this PR.

Reply:
Okay, I moved the `impl` crumb from xsCollections to `MacroXSGenerationInterface.buildMacros())`.